web_src/js/index.js

```diff
@@ -2421,7 +2421,7 @@ function u2fError(errorType) {
   u2fErrors[errorType].removeClass('hide');
 
   Object.keys(u2fErrors).forEach((type) => {
-    if (type !== errorType) {
+    if (type !== errorType.toString()) {
```

Comment:
For some reason, I have a feeling that:

```suggestion
    if (type !== `${errorType}`) {
```

Would be better mainly because it's protected against nulls but I'm not certain that's a real issue here.

Reply:
Updated :+1: